pkg/controller/managementingress/handler/configmap.go

```diff
@@ -146,6 +147,53 @@ func (ingressRequest *IngressRequest) CreateOrUpdateConfigMap() error {
 		return fmt.Errorf("Failure creating bind info for %q: %v", ingressRequest.managementIngress.Name, err)
 	}
 
+	// create configmap ibmcloud-cluster-info
+	baseDomain, err := ingressRequest.GetRouteAppDomain()
+	if err != nil {
+		return fmt.Errorf("Failure getting route base domain %q: %v", ingressRequest.managementIngress.Name, err)
+	}
+
+	ver := os.Getenv(CSVersion)
+	if ver == "" {
+		ver = CSVersionValue
+	}
+	cname := os.Getenv(ClusterName)
+	if cname == "" {
+		cname = ClusterNameValue
+	}
+
+	rhttpPort := os.Getenv(RouteHTTPPort)
```

Comment:
For the env variable I think we should have different name with upper case. For example, ROUTE_HTTP_PORT, etc.